text/0097-log-data-model.md

```diff
@@ -1002,6 +1003,326 @@ span_id          | string             | The span ID within the trace associated
 labels           | map<string,string> | A set of user-defined (key, value) data that provides additional information about the log entry. | Attributes
 All other fields |                    |                                                         | Attributes["google.*"]
 
+
+## Elastic Common Schema
+
+<table>
+  <tr>
+    <td>Field</td>
+    <td>Type</td>
+    <td>Description</td>
+    <td>Maps to Unified Model Field</td>
+  </tr>
+  <tr>
+    <td>@timestamp</td>
+    <td>datetime</td>
+    <td>Time the event was recorded</td>
+    <td>timestamp</td>
+  </tr>
+  <tr>
+    <td>message</td>
+    <td>string</td>
+    <td>Any type of message</td>
+    <td>body</td>
+  </tr>
+  <tr>
+    <td>labels</td>
+    <td>key/value</td>
+    <td>Arbitrary labels related to the event</td>
+    <td>attributes[*]</td>
+  </tr>
+  <tr>
+    <td>tags</td>
+    <td>array of string</td>
+    <td>List of values related to the event</td>
+    <td>?</td>
+  </tr>
+  <tr>
+    <td>trace.id</td>
+    <td>string</td>
+    <td>Trace ID</td>
+    <td>trace_id</td>
+  </tr>
+  <tr>
+    <td>span.id*</td>
+    <td>string</td>
+    <td>Span ID</td>
+    <td>span_id</td>
+  </tr>
+  <tr>
+    <td>agent.ephemeral_id</td>
+    <td>string</td>
+    <td>Ephemeral ID created by agent</td>
+    <td>**resource</td>
+  </tr>
+  <tr>
+    <td>agent.id</td>
+    <td>string</td>
+    <td>Unique identifier of this agent</td>
+    <td>**resource</td>
+  </tr>
+  <tr>
+    <td>agent.name</td>
+    <td>string</td>
+    <td>Name given to the agent</td>
+    <td>resource["telemetry.sdk.name"]</td>
+  </tr>
+  <tr>
+    <td>agent.type</td>
+    <td>string</td>
+    <td>Type of agent</td>
+    <td>resource[“telemetry.sdk.language”]</td>
+  </tr>
+  <tr>
+    <td>agent.version</td>
+    <td>string</td>
+    <td>Version of agent</td>
+    <td>resource[“telemetry.sdk.version”]</td>
+  </tr>
+  <tr>
+    <td>source.ip, client.ip</td>
+    <td>string</td>
+    <td>The IP address that the request was made from.</td>
+    <td>resource[“net.peer.ip”] or resource[“net.host.ip”]</td>
```

Comment:
Judging from the description this can be different for different log records coming from the same resource. If that is true it should be in `attributes` not in `resource`.